recipe/recipe.yaml

```diff
@@ -29,10 +34,18 @@ requirements:
         - m2-findutils
         - m2-sed
   host:
-    - dotnet
+    - if: not unix
+      then:
+        - dotnet >=8.0.302
+      else:
+        - dotnet >=8.0.302
     - renode-cores ==${{ version }}
   run:
-    - dotnet
+    - if: not unix
+      then:
+        - dotnet-desktop >=8.0.6
+      else:
+        - dotnet-runtime >=8.0.6
```

Comment:
Was there a change such that the `dotnet` package doesn't provide these anymore?

Reply:
`dotnet` includes the whole SDK, so this is a more minimal inclusion and probably preferable.

Reply:
But yes, `dotnet` still includes both of these.

Reply:
Confirming that I did that for 'minimal specific inclusion'

Reply:
Reverted to dotnet